metadata_tools/util.py

```
@@ -4,12 +4,44 @@
+def PdsTable(label_path):
+    """read a pds3table from an FCPath object.  To be replaced whenever pdstable is 
+       upgraded to use filecache.
+
+    Args:
+        label_path (str, Path, or FCPath): Path to the label file.
+
+    Returns:
+        pdstable.PdsTable: Table associated with the given label.
+    """
+    local_label_path = label_path.retrieve()
+    local_table_path = label_path.with_suffix('.tab').retrieve() # Retrieve table as well
```

Flake8 (7.3.0)

[error] local variable 'local_table_path' is assigned to but never used

(F841)